scss/_tables.scss

```diff
@@ -99,8 +99,10 @@
 
 .table-striped {
   > tbody > tr:nth-of-type(#{$table-striped-order}) {
-    --bs-table-accent-bg: var(--bs-table-striped-bg);
-    color: var(--bs-table-striped-color);
+    color: $table-striped-color;
+    > * {
+      background-image: linear-gradient($table-striped-bg, $table-striped-bg);
+    }
```

Comment:
We avoid here usage of custom properties. It is only possible during variants / modifiers generation. So that we can backport to IE11 here (except variants)

Also it is easier to reason about:
We had `--bs-table-accent-bg: var(--bs-table-striped-bg);` not clear what it changes, it changes accent color ? Need to Ctrl + F what exactly it will change and it will be easy to make a mistake when we will have more variables and more selectors

As a downside - yes, more code